actions.py

```diff
@@ -61,17 +71,25 @@ def album_search(artist, album, media, format):
 			album = {"action": "torrentgroup", "id": str(group["groupId"])}
 			r2 = make_request(album).json()["response"]
 			for release in r2["torrents"]:
-				if release["format"].lower() in format and release["media"].lower() in media:
-					if release["encoding"] == "24bit Lossless":
-						print("***THIS IS A 24-BIT RELEASE***")
-					print("Torrent ID: " + str(release["id"]))
-					print("Media: " + release["media"])
-					print("Size: " + str(sizeof_fmt(release["size"])))
-					print("Files: " + str(release["fileCount"]))
-					print("Seeders: " + str(release["seeders"]))
-					print("")
+				if format == None and media == None:
```

Comment:
Here you test for the 4 possibilities of both `format` and `media` boolean values...this can probably be simplified. I'll try to look into it. 🤔